fixtures/uking/mini-spider-light.json

```diff
@@ -0,0 +1,317 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Mini Spider Light",
+  "shortName": "UK-MiniSpider",
+  "categories": ["Effect"],
+  "meta": {
+    "authors": ["Anonymus", "CGBassPlayer"],
+    "createDate": "2023-03-21",
+    "lastModifyDate": "2023-04-24"
+  },
+  "comment": "Mini 60W Spider Light",
```

Comment:
The product page mentions 80W. But since this information is collected separately and the rest is just the name, I'd say let's remove the comment completely.

```suggestion
```

Reply:
I actually own this fixture and I still could not tell you what the actual wattage of the LEDs are. It seems like its different everywhere you go